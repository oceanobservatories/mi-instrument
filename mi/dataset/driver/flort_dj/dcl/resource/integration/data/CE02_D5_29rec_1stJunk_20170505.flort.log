2017/05/05 21:45:13.971 [flort:DLOGP2]:Instrument sampling - Data logger entering data acquisition state
2017/05/05 21:45:14.008 05/17	21:43:58	700	259	695	77	460	64	556
2017/05/05 21:45:15.136 05/05/17	21:43:59	700	249	695	77	460	66	556
2017/05/05 21:45:16.264 05/05/17	21:44:00	700	228	695	78	460	66	556
2017/05/05 21:45:28.676 05/05/17	21:44:13	700	292	695	81	460	65	555
2017/05/05 21:45:51.248 05/05/17	21:44:35	700	265	695	72	460	65	554
2017/05/05 21:45:52.377 05/05/17	21:44:36	700	273	695	79	460	65	554
2017/05/05 21:45:53.505 05/05/17	21:44:37	700	293	695	76	460	66	554
2017/05/05 21:46:00.276 05/05/17	21:44:44	700	237	695	81	460	66	554
2017/05/05 21:46:34.131 05/05/17	21:45:18	700	273	695	71	460	62	553
2017/05/05 21:46:35.259 05/05/17	21:45:19	700	241	695	74	460	63	553
2017/05/05 21:46:36.388 05/05/17	21:45:20	700	241	695	77	460	61	553
2017/05/05 21:46:37.516 05/05/17	21:45:21	700	238	695	72	460	64	553
2017/05/05 21:46:38.645 05/05/17	21:45:23	700	297	695	78	460	65	553
2017/05/05 21:46:39.774 05/05/17	21:45:24	700	309	695	75	460	61	553
2017/05/05 21:46:40.905 05/05/17	21:45:25	700	244	695	78	460	66	553
2017/05/05 21:46:47.673 05/05/17	21:45:32	700	256	695	72	460	66	553
2017/05/05 21:46:48.802 05/05/17	21:45:33	700	275	695	74	460	66	553
2017/05/05 21:47:22.677 05/05/17	21:46:07	700	250	695	75	460	60	552
2017/05/05 21:47:23.785 05/05/17	21:46:08	700	244	695	77	460	66	552
2017/05/05 21:47:24.913 05/05/17	21:46:09	700	274	695	76	460	63	552
2017/05/05 21:47:26.042 05/05/17	21:46:10	700	293	695	75	460	64	552
2017/05/05 21:47:27.171 05/05/17	21:46:11	700	299	695	77	460	67	552
2017/05/05 21:47:28.299 05/05/17	21:46:12	700	263	695	78	460	64	552
2017/05/05 21:47:35.071 05/05/17	21:46:19	700	227	695	79	460	65	551
2017/05/05 21:47:36.199 05/05/17	21:46:20	700	400	695	75	460	66	552
2017/05/05 21:47:37.328 05/05/17	21:46:21	700	341	695	81	460	67	552
2017/05/05 21:47:59.899 05/05/17	21:46:44	700	300	695	80	460	64	551
2017/05/05 21:48:01.028 05/05/17	21:46:45	700	293	695	80	460	64	551
2017/05/05 21:48:01.235 [flort:DLOGP2]:Instrument Stopped [Power Off]
