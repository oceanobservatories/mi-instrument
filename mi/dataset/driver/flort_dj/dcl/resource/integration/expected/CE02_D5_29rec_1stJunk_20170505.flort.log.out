
 subsite  | node  | sensor       | bin        | method      | time                  | deployment | internal_timestamp | port_timestamp        | preferred_timestamp | quality_flag | date_string | dcl_controller_timestamp | measurement_wavelength_beta | measurement_wavelength_cdom | measurement_wavelength_chl | raw_internal_temp | raw_signal_beta | raw_signal_cdom | raw_signal_chl | time_string
----------+-------+--------------+------------+-------------+-----------------------+------------+--------------------+-----------------------+---------------------+--------------+-------------+--------------------------+-----------------------------+-----------------------------+----------------------------+-------------------+-----------------+-----------------+----------------+-------------
 CE02SHSM | RID27 | 02-FLORTD000 | 3702931200 | telemetered | 3703009515.1359996796 |          5 |         3703009439 | 3703009515.1359996796 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               556 |             249 |              66 |             77 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702931200 | telemetered |  3703009516.263999939 |          5 |         3703009440 |  3703009516.263999939 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               556 |             228 |              66 |             78 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702931200 | telemetered | 3703009528.6760001183 |          5 |         3703009453 | 3703009528.6760001183 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               555 |             292 |              65 |             81 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702931200 | telemetered |  3703009551.248000145 |          5 |         3703009475 |  3703009551.248000145 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               554 |             265 |              65 |             72 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702931200 | telemetered |  3703009552.376999855 |          5 |         3703009476 |  3703009552.376999855 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               554 |             273 |              65 |             79 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702931200 | telemetered | 3703009553.5050001144 |          5 |         3703009477 | 3703009553.5050001144 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               554 |             293 |              66 |             76 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702931200 | telemetered | 3703009560.2760000229 |          5 |         3703009484 | 3703009560.2760000229 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               554 |             237 |              66 |             81 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702931200 | telemetered |  3703009594.131000042 |          5 |         3703009518 |  3703009594.131000042 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             273 |              62 |             71 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702931200 | telemetered | 3703009595.2589998245 |          5 |         3703009519 | 3703009595.2589998245 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             241 |              63 |             74 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702931200 | telemetered | 3703009596.3880000114 |          5 |         3703009520 | 3703009596.3880000114 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             241 |              61 |             77 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702931200 | telemetered |  3703009597.515999794 |          5 |         3703009521 |  3703009597.515999794 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             238 |              64 |             72 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702931200 | telemetered | 3703009598.6449999809 |          5 |         3703009523 | 3703009598.6449999809 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             297 |              65 |             78 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702931200 | telemetered | 3703009599.7740001678 |          5 |         3703009524 | 3703009599.7740001678 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             309 |              61 |             75 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702931200 | telemetered |  3703009600.904999733 |          5 |         3703009525 |  3703009600.904999733 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             244 |              66 |             78 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702931200 | telemetered | 3703009607.6730003357 |          5 |         3703009532 | 3703009607.6730003357 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             256 |              66 |             72 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702931200 | telemetered | 3703009608.8020000458 |          5 |         3703009533 | 3703009608.8020000458 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             275 |              66 |             74 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702931200 | telemetered | 3703009642.6770000458 |          5 |         3703009567 | 3703009642.6770000458 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               552 |             250 |              60 |             75 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702931200 | telemetered | 3703009643.7849998474 |          5 |         3703009568 | 3703009643.7849998474 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               552 |             244 |              66 |             77 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702931200 | telemetered | 3703009644.9130001068 |          5 |         3703009569 | 3703009644.9130001068 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               552 |             274 |              63 |             76 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702931200 | telemetered | 3703009646.0419998169 |          5 |         3703009570 | 3703009646.0419998169 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               552 |             293 |              64 |             75 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702931200 | telemetered | 3703009647.1710000038 |          5 |         3703009571 | 3703009647.1710000038 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               552 |             299 |              67 |             77 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702931200 | telemetered | 3703009648.2989997864 |          5 |         3703009572 | 3703009648.2989997864 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               552 |             263 |              64 |             78 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702931200 | telemetered | 3703009655.0710000992 |          5 |         3703009579 | 3703009655.0710000992 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               551 |             227 |              65 |             79 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702931200 | telemetered | 3703009656.1989998817 |          5 |         3703009580 | 3703009656.1989998817 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               552 |             400 |              66 |             75 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702931200 | telemetered | 3703009657.3280000687 |          5 |         3703009581 | 3703009657.3280000687 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               552 |             341 |              67 |             81 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702931200 | telemetered | 3703009679.8990001678 |          5 |         3703009604 | 3703009679.8990001678 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               551 |             300 |              64 |             80 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702931200 | telemetered | 3703009681.0279998779 |          5 |         3703009605 | 3703009681.0279998779 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               551 |             293 |              64 |             80 |        null

(27 rows)
